Resources/ServerInfo/Guidebook/Engineering/DeviceMonitoringAndControl.xml

```diff
@@ -20,4 +22,53 @@
 
   If you need to clear your tool, you can press (replace with fancy interact control key puller) on the tool to bring up a list of saved devices, and then press the "Clear" button.
 
+  ## User Interface
+  Interacting with an Air Alarm will bring up a menu with 3 sections:
+  - Top: The current status of the air alarm. This includes the current pressure and temperature of the area the air alarm is monitoring.
+  - Middle: Lists of linked air vents and air scrubbers, as well as all sensors reporting back to the air alarm.
+  - Bottom: Buttons to control air alarm operating modes, as well as the ability to toggle the automatic changing of modes.
+
+  ## Alarm Status
+  Air alarms quickly communicate the status of the area they are monitoring through the color of the light on the device:
+  - [color=green]Normal[/color]: The area is safe to breathe in.
+  - [color=yellow]Warning[/color]: The area is starting to deviate from the desired atmosphere, and is becoming unsafe.
+  - [color=red]Danger[/color]: The area is no longer safe, internals and/or hardsuits are required.
+
+  Air alarms will automatically drop (close) linked firelocks when the [color=orange]Warning[/color] status is reached. These firelocks will automatically raise (open) when the status returns to [color=green]Normal[/color].
+
+  Air alarms have Signal outputs for each of the three statuses, which can be used to trigger other signal devices when the status changes.
+
+  ## Operating Modes
```

Comment:
you forgot to explain sensors, how air vents and scrubbers also have sensors, and how they tie into statuses you moron